.eslintrc.cjs

```diff
@@ -14,7 +19,7 @@ module.exports = {
   },
   parserOptions: {
     sourceType: "module",
-    ecmaVersion: 2019,
+    ecmaVersion: 2020,
```

Comment:
Why this change?

Reply:
Just took this from the new template: https://github.com/sveltejs/kit/blob/master/packages/create-svelte/shared/%2Beslint%2Bprettier%2Btypescript/.eslintrc.cjs